.github/workflows/unit_test.yml

```
@@ -44,7 +44,7 @@ jobs:
         run: poetry install --no-interaction --no-root
         if: steps.cache.outputs.cache-hit != 'true'
       - name: Install project
-        run: poetry install --no-interaction --with test
+        run: poetry install --no-interaction --with test --all-extras
```

```suggestion
        run: poetry install --no-interaction --with test
```
this is not strictly necessary anymore I believe, as we added the extras to the test dependencies